### DISCUSSION TITLE
Personal notes that can be easily chosen to be exported or not (like presenter's notes in Beamer)

### DISCUSSION THREAD

**Discussion by hao-hong** [quarto-dev/quarto-cli#3063] [Q&A]
Hi all, 

Suppose that I am writing an article and will export it into a .pdf file. Is there a way to easily include some personal notes, so that I can choose whether to include them in the exported .pdf file? (something similar to the "presenter's notes" in Beamer)

**Comment by mcanouil** [comment]
You could use the profile feature to set a profile "article" and "article-with-notes".
See <https://quarto.org/docs/projects/profiles.html>.
